src/schemas/execute.yaml

```
@@ -0,0 +1,277 @@
+      title: Nonce
+      $ref: '#/components/schemas/uint64'
+    balance:
+      title: Balance
+      $ref: '#/components/schemas/uint256'
+    code:
+      title: Code
+      $ref: '#/components/schemas/bytes'
+    state:
+      title: Storage
+      $ref: '#/components/schemas/AccountStorage'
+AccountOverrideStateDiff:
+  title: Account override with partial storage modification
+  required:
+    - stateDiff
+  properties:
+    address:
+      title: Address
+      $ref: '#/components/schemas/address'
+    nonce:
+      title: Nonce
+      $ref: '#/components/schemas/uint64'
+    balance:
+      title: Balance
+      $ref: '#/components/schemas/uint256'
+    code:
+      title: Code
+      $ref: '#/components/schemas/bytes'
+    stateDiff:
+      title: Storage difference
+      $ref: '#/components/schemas/AccountStorage'
+AccountStorage:
+  title: Storage slots for an account
+  type: object
+  additionalProperties:
+    - $ref: '#/components/schemas/hash32'
+BlockOverride:
+  title: Context fields related to the block being executed
+  type: object
+  properties:
+    number:
+      title: Number
+      $ref: '#/components/schemas/uint256'
+    prevRandao:
+      title: Randomness beacon
+      $ref: '#/components/schemas/uint256'
+    time:
+      title: Time
+      $ref: '#/components/schemas/uint256'
+    gasLimit:
+      title: Gas limit
+      $ref: '#/components/schemas/uint64'
+    feeRecipient:
+      title: feeRecipient
+      $ref: '#/components/schemas/address'
+    baseFee:
+      title: Base fee
+      $ref: '#/components/schemas/uint256'
+MultiCallResult:
+  title: Full results of multi call
+  type: array
+  items:
+    $ref: '#/components/schemas/MultiCallBlockResult'
+MultiCallBlockResult:
+  title: Result of multicall block-level, with array of calls
+  type: object
+  properties:
+    number:
+      title: Number
+      $ref: '#/components/schemas/uint256'
+    hash:
+      title: block hash
+      $ref: '#/components/schemas/hash32'
+    timestamp:
+      title: Time
+      $ref: '#/components/schemas/uint256'
+    gasLimit:
+      title: Gas limit
+      $ref: '#/components/schemas/uint64'
+    gasUsed:
+      title: Gas used
+      $ref: '#/components/schemas/uint64'
+    feeRecipient:
+      title: feeRecipient
+      $ref: '#/components/schemas/address'
+    baseFeePerGas:
+      title: Base fee
+      $ref: '#/components/schemas/uint256'
+    calls:
+      $ref: '#/components/schemas/CallResults'
+CallResults:
+  title: Results of multi call within block
+  type: array
+  items:
+    anyOf:
+      - $ref: '#/components/schemas/CallResultFailure'
+      - $ref: '#/components/schemas/CallResultSuccess'
+      - $ref: '#/components/schemas/CallResultInvalid'
+CallResultFailure:
+  title: Result of call failure
+  type: object
+  required:
+    - status
+    - return
+    - error
+    - gasUsed
+  properties:
+    status:
+      title: Call Status Failure
+      type: string
+      pattern: ^0x0$
+    return:
```

```suggestion
    returnValue:
```
return is often reserved word, so I would not use it

---

Modified to `returnData`